ONSdigital/eq-questionnaire-runner
# Cache questionnaire schemas on app start up

### What is the context of this PR?
- To potentially reduce some higher than normal response times due to cold starts, this PR preloads the questionnaire schema caches on app start up instead of caching the schema on the first request to a given schema. Only schemas in the `schemas` dir is cached on start up.
- Added some extra tests for loading schemas that was failing coverage on this branch.
- Removed wrapper method used in tests which was identical to `load_schema_from_name`.

### How to review 
Ensure the all schemas including their language variants are cached on app start up.

To test you can either:
- load the schema from launcher and ensure the `loading schema` log line is not present when switching between different schemas.
- debug the app and check the cache size and hit ratio using [CacheInfo](https://docs.python.org/3/library/functools.html#functools.lru_cache)

### Checklist
